src/Css/Specificity/Specificity.php

```diff
@@ -80,8 +92,8 @@ public function getValues()
      */
     public static function fromSelector($selector)
     {
-        $pattern_a = "  \#";
-        $pattern_b = "  (\.[\w]+)                     # classes
+        $idSelectorsPattern = "  \#";
```

Comment:
This is just ridiculous, why on earth would you want to use classnames like this... I will mention in the readme it won't support css-escapes...